shell/common/shell.cc

```diff
@@ -369,6 +369,21 @@ Shell::~Shell() {
   platform_latch.Wait();
 }
 
+void Shell::PurgeCaches() const {
+  task_runners_.GetGPUTaskRunner()->PostTask(
+      [rasterizer = rasterizer_->GetWeakPtr()]() {
+        if (rasterizer) {
+          rasterizer->PurgeCaches();
+        }
+      });
+  task_runners_.GetIOTaskRunner()->PostTask(
```

Comment:
This is redundant since the IO manager GrContext caches are disabled. https://github.com/flutter/engine/blob/aa9b3a180a114b3cc917694a72f6896f9dc013c3/shell/common/shell_io_manager.cc#L38

Reply:
Removed and added a comment for future me